example/spec/standalone/redeployment/wait_for_deploy_without_hello_world_workflow.tf

```diff
@@ -0,0 +1,35 @@
+module "wait_for_deploy_workflow" {
```

Comment:
Rather than load the workflow from the config file, remove a task, and write the updated workflow back out to the config file, I'm just overwriting the existing config with this config, which does not have the "Hello World" step.